File: javalin-openapi-processor/src/main/java/net/dzikoysk/openapi/processor/processing/ClassVisitor.java
package net.dzikoysk.openapi.processor.processing;

import javax.lang.model.type.TypeMirror;

public final class ClassVisitor extends DefaultVisitor<TypeMirror, Void> {

    @Override
    public TypeMirror visitType(TypeMirror typeMirror, Void unused) {
        return typeMirror;
    }

}

File: openapi-annotation-processor/src/main/kotlin/io/javalin/openapi/processor/processing/ClassVisitor.kt
package io.javalin.openapi.processor.processing

import javax.lang.model.type.TypeMirror

internal class ClassVisitor : DefaultVisitor<TypeMirror, Void?>() {

    @Suppress("PARAMETER_NAME_CHANGED_ON_OVERRIDE")
    override fun visitType(typeMirror: TypeMirror, unused: Void?): TypeMirror = typeMirror

}